packages/rest-accounts-bearer-token/accounts_bearer_token.js

```
@@ -0,0 +1,60 @@
+var Fiber = Npm.require("fibers");
+
+// Add the authorized user ID, if any, to the request
+JsonRoutes.middleWare.use(function (req, res, next) {
+  Fiber(function () {
+    var userId = getUserIdFromRequest(req);
+    if (userId) {
+      req.userId = userId;
+    }
+    next();
+  }).run();
+});
+
+// Get the user ID from the standard bearer token authorization header
+function getUserIdFromRequest(req) {
+  if (! _.has(Package, "accounts-base")) {
+    return null;
+  }
+
+  // Looks like "Authorization: Bearer <token>"
+  var token = req.headers.authorization &&
+    req.headers.authorization.split(" ")[1];
+
+  if (! token) {
+    return null;
+  }
+
+  // Check token expiration
+  var tokenExpires = Package["accounts-base"].Accounts._tokenExpiration(token.when);
+  if (new Date() >= tokenExpires) {
+    throw new Meteor.Error("token-expired", "Your login token has expired. Please log in again.");
```

I don't think we can simply throw an error here anymore. We'll have to respond with 401 error.